scss/_base_forms.scss

```
@@ -162,6 +162,12 @@
     }
   }
 
+  [type='text'] {
```

I'm not sure we want to limit it to text type, we can imagine it being used with numbers, URLs, etc.
But I'd limit it to inputs with `list` attribute.

```suggestion
  input[list] {
```

But verify that this works, I haven't check it ;)

---

Added, and it works!